Gopkg.lock

```diff
@@ -22,7 +22,7 @@
   version = "v0.2.0"
 
 [[projects]]
-  digest = "1:355d491068cd888a3725d68cd09681a49251d6d8e3cb49120bf6b83a86c7ee0f"
+  digest = "1:c7ec3a8daf56c9d7c4bb36f498f6db3f8c339b1142300dcd4328950574f91c6f"
```

Comment:
Was updating this file intentional?

Reply:
This file was updated by hack/update-deps.sh (dep ensure). First version of PR didn't have that change and failed on integration test(please see Prow history).